lk2nd/device/dts/msm8974/samsung.dts

```diff
@@ -38,6 +38,32 @@
 		};
 	};
 
+	hltetmo {
+		model = "Samsung Galaxy Note 3 (SM-N900T)";
+		compatible = "samsung,hlte";
+		qcom,msm-id = <0x7E01FF01 7 0>;
```

Comment:
Are you sure the original dtb in downstream boot.img uses rev7? (i.e. if you decompile all dtbs and check for "h project, would "rev 7" be the latest one?)